src/lib.rs

```diff
@@ -1137,4 +1137,14 @@ mod tests {
             &[],
         );
     }
+
+    #[test]
```

Comment:
Can we also add a test for using "self" in the subject of an allow rule?  It should already be handled by the validate_arguments() code, but it would be nice to have some additional verification.